_assets/stylesheets/layout/_menu.scss

```diff
@@ -100,4 +100,13 @@
 				}
 			}
 		}
+
+		.stack-info {

```

Comment:
Let's move this class into `#sidebar` on `_assets/stylesheets/layout/_sidebar.scss` since this will only apply to the sidebar, similar to the vars.

Reply:
done! :)